operator/webhooks/pod_mutator/pod_mutating_webhook.go

```diff
@@ -126,12 +126,6 @@ func (a *PodMutatingWebhook) Handle(ctx context.Context, req admission.Request)
 // PodMutatingWebhook implements admission.DecoderInjector.
```

Comment:
Can we please remove also this comment ? :)

Reply:
Sure will do that now